Lib/fontTools/varLib/models.py

```diff
@@ -379,9 +379,9 @@ def getDeltas(self, masterValues, *, round=noRound):
 			out.append(round(delta))
 		return out
 
-	def getDeltasAndSupports(self, items, *, round=noRound):
+	def getDeltasAndSupports(self, items, **kwargs):
```

Comment:
i don't like too much residual `**kwargs` at the end of function signatures, they make the API less clear. I think it was better with the keyword-only `round` parameter

Reply:
I don't like them either. I did when I was changing the default round and realized that the default was hardcoded in 4 places... Up to you.